File: video-series-mono/src/main/java/org/jesperancinha/video/VsaMonoApplicationLauncher.java
package org.jesperancinha.video;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.boot.autoconfigure.jdbc.DataSourceAutoConfiguration;
import org.springframework.boot.autoconfigure.web.servlet.error.ErrorMvcAutoConfiguration;

@SpringBootApplication(exclude = {
        ErrorMvcAutoConfiguration.class,
        DataSourceAutoConfiguration.class
})
public class VsaMonoApplicationLauncher {
    public static void main(String[] args) {
        SpringApplication.run(VsaMonoApplicationLauncher.class, args);
    }
}

File: video-series-mono/src/main/kotlin/org/jesperancinha/video/VsaMonoApplicationLauncher.kt
package org.jesperancinha.video

import org.springframework.boot.SpringApplication
import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.jdbc.autoconfigure.DataSourceAutoConfiguration
import org.springframework.boot.webmvc.autoconfigure.error.ErrorMvcAutoConfiguration

@SpringBootApplication(exclude = [ErrorMvcAutoConfiguration::class, DataSourceAutoConfiguration::class])
class VsaMonoApplicationLauncher {
    companion object {
        @JvmStatic
        fun main(args: Array<String>) {
            SpringApplication.run(VsaMonoApplicationLauncher::class.java, *args)
        }
    }
}